index.html

```diff
@@ -58,24 +45,17 @@ <h1>heading1</h1>
                             natus incidunt facilis odio non rerum.
                         </p>
                     </div>
-                    <div class="mugshot">
-                        <div class="paperclip"></div>
-                        <a href="links.html">
-                            <img src="Sault%20College%20Logo%20Transparent.png" alt="Sault College Logo" />
-                            <span>Sault College Logo?</span>
-                        </a>
-                    </div>
                 </div>
             </section>
-            <section class="bio">
+            <section class="favourites">
                 <h2>My Three Favorite ball games Basketball, football and tennis</h2>
                 <ol>
                     <li>Basketball</li>
                     <li>football</li>
                     <li>tennis</li>
                 </ol>
             </section>
-            <section class="bio">
+            <section class="haiku">
                 <h2 style="color: #b400ff;">Haiku</h2>
```

Comment:
Using the same class `bio` on all these elements was causing issues with your grid layout